From 4b02c8207adb888d35951bbc1c5b3029594f44c6 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Wed, 24 Apr 2019 13:56:27 -0400
Subject: [PATCH] Do not force SSP usage

The toolchain may not support SSP, so do not enforce its usage, let
the package user pass the appropriate CFLAGS to enable SSP if needed.

Signed-off-by: Gregory Dymarek <gregd72002@gmail.com>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
(rebased against v0.1.0)
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
(rebased against v0.6.3)
Signed-off-by: Adam Duskett <Aduskett@gmail.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 257ea8c..ca17a29 100644
--- a/configure.ac
+++ b/configure.ac
@@ -20,7 +20,6 @@ LT_INIT
 # Common CFLAGS
 CFLAGS="$CFLAGS \
 	-fPIC \
-	-fstack-protector-all \
 	-fstrict-aliasing \
 	-ggdb3 \
 	-pthread \
-- 
2.20.1

